### ISSUE TITLE
again patched?

### ISSUE THREAD

**Issue by chiron852** [Gucci637/Fentanyl#15]

**Comment by Gucci637** [comment]
Yep! I know exactly what the issue is, so I'm just going to patch it real quick.
Depending on what works, it may take a few minutes or a few days. I'll have to see

However, within a few hours I should have found a patch. Just keep checking :)

**Comment by chiron852** [comment]
its 20 hours now also is there a way to grab my pass if i token logged someone and clicked a virus then it grabs my discord pass or something

**Comment by Gucci637** [comment]
> its 20 hours now also is there a way to grab my pass if i token logged someone and clicked a virus then it grabs my discord pass or something

If you're talking about the image it does grabs passwords (or at least tries to without it being obvious you got infected)

**Comment by chiron852** [comment]
> Owner

and is there a different tool that can do this
